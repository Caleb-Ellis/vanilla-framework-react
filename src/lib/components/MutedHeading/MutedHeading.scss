@import 'vanilla-framework/scss/base';
@import 'vanilla-framework/scss/patterns_muted-heading';
@include vf-b-typography;
@include vf-p-muted-heading;
