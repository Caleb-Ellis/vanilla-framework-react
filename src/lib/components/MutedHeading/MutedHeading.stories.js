import React from 'react';
import { storiesOf } from '@storybook/react';
import { withKnobs, text } from '@storybook/addon-knobs';
import { withInfo } from '@storybook/addon-info';

import MutedHeading from './MutedHeading';

storiesOf('Muted Heading', module).addDecorator(withKnobs)
  .add('Default',
    withInfo('The MutedHeading component can be used to introduce a collection of icons or images.')(() => (
      <MutedHeading>{text('Text', 'Muted heading')}</MutedHeading>),
    ),
  );
