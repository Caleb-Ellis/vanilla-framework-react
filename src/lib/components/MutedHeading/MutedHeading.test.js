import React from 'react';
import ReactTestRenderer from 'react-test-renderer';
import MutedHeading from './MutedHeading';

describe('MutedHeading component', () => {
  it('should render correctly', () => {
    const mutedHeading = ReactTestRenderer.create(
      <MutedHeading>Muted heading</MutedHeading>);
    const json = mutedHeading.toJSON();
    expect(json).toMatchSnapshot();
  });
});
