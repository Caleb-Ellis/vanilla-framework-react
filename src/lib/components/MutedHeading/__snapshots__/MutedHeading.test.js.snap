// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MutedHeading component should render correctly 1`] = `
<h4
  className="p-muted-heading"
>
  Muted heading
</h4>
`;
